pkgs/development/haskell-modules/configuration-ghc-8.6.x.nix

```diff
@@ -91,7 +95,7 @@ self: super: {
   ghc-lib-parser-ex = addBuildDepend self.ghc-lib-parser super.ghc-lib-parser-ex;
 
   # This became a core library in ghc 8.10., so we don’t have an "exception" attribute anymore.
-  exceptions = super.exceptions_0_10_4;
+  exceptions = null;
```

Comment:
We could still build a version of exceptions, but I suppose it hardly matters here.

Reply:
Good point! Let's restore it for consistency. Used `0.10.7` in proposed https://github.com/NixOS/nixpkgs/pull/278310